tests/x11/start_wayland_plasma5.pm

```diff
@@ -45,14 +45,24 @@ sub run {
     send_key 'ret';
 
     # Wait until logged in
-    assert_screen 'generic-desktop', 60;
-
-    # We're now in a wayland session, which is in a different VT
-    x11_start_program('xterm');
-    my $tty = script_output('echo $XDG_VTNR');
-    send_key("alt-f4");                                # close xterm
+    wait_still_screen;
```

Comment:
No, because we will have a race condition looking for SDDM just after typing the password. There are some miliseconds of delay until the black screen.

Reply:
But please do not use the insane default of 7 seconds here!